woocommerce/woocommerce
# Change the Woo logo in the marketing banner

### Changes proposed in this Pull Request:

This pull request is a follow-up of https://github.com/woocommerce/woocommerce/pull/53308, it changes the Woo logo in the Marketing banner and also changes the purple color of the other icons in the banner:

![image](https://github.com/user-attachments/assets/6076d1cc-a4b3-4185-b7ac-a1cce3cc68f5)


### How to test the changes in this Pull Request:

1. Activate the new branding as instructed in https://github.com/woocommerce/woocommerce/pull/53308
2. Go to Marketing - Overview and verify that the new Woo logo appears. also the colors of the other icons in the banner match the color of the new logo.
3. Deactivate the new branding, reload the page, and verify that you see the old logo and the old colors again.

### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
